lib/kms.js

```diff
@@ -37,12 +62,20 @@ export async function createKeystore({
     keystoreConfig.referenceId = referenceId;
   }
   const {httpsAgent} = brHttpsAgent;
-  return KmsClient.createKeystore({
-    url: `${bedrock.config.tokenizer.kms.baseUrl}/keystores`,
-    config: keystoreConfig,
-    invocationSigner,
-    httpsAgent
-  });
+  let result;
+  try {
+    result = await KmsClient.createKeystore({
+      url: `${bedrock.config.tokenizer.kms.baseUrl}/keystores`,
+      config: keystoreConfig,
+      invocationSigner,
+      httpsAgent
+    });
+  } catch(cause) {
+    throw new BedrockError(
+      'Error creating keystore for tokenizer.', 'UnknownError',
+      {httpStatusCode: 400, public: true}, cause);
```

Comment:
This is a server-side error, right?

```suggestion
      {httpStatusCode: 500, public: true}, cause);
```

Reply:
We also probably need to log this.

Reply:
Yep, logging.